server/controllers/auth/login.js

```diff
@@ -0,0 +1,3 @@
+exports.login = (req, res) => {
```

Comment:
i think we deal to add the `next` for all routes
 so we can handle the error at once in the `error` middleware

Reply:
done